src/code/graph.c

```diff
@@ -31,6 +31,9 @@ void Graph_FaultClient(void) {
     osViSwapBuffer(nextFb);
 }
 
+// TODO: merge Gfx and GfxMod to make this function's arguments consistent
+void UCodeDisas_Disassemble(UCodeDisas*, Gfx*);
```

Comment:
Why put this here and not in `functions.h` ? (note `UCodeDisas_Disassemble` is commented out in `functions.h`)

Reply:
The function takes a `GfxMod*` in `ucode_disas.c` so it can't be defined as taking `Gfx*` in `functions.h` yet. This is why I added a TODO so we don't forget to fix it.